.gitignore

```diff
@@ -390,5 +390,8 @@ $RECYCLE.BIN/
 # vsCode
 .vscode
 
+# Python venv
+venv/
+
```

Comment:
You shouldn't put your Virtual Environment in the same directory as it would be package with the code for each test which slows the test down. Please remove this addition.